### PR TITLE
default to verbose param display

### PR DESCRIPTION
**Pull request by tridge**

### CODE REVIEW

File: MAVProxy/modules/lib/param_help.py [hunk 1]
```diff
@@ -31,26 +31,30 @@ def param_help_download(self):
...
+                    print("Unknown vehicle type")
                 return None
             path = mp_util.dot_mavproxy("%s.xml" % self.vehicle_name)
             if not os.path.exists(path):
                 if self.vehicle_name == 'APMrover2':
                     path = mp_util.dot_mavproxy("%s.xml" % "Rover")
             if not os.path.exists(path):
-                print("Please run 'param download' first (vehicle_name=%s)" % self.vehicle_name)
+                if verbose:
+                    print("Please run 'param download' first (vehicle_name=%s)" % self.vehicle_name)
```
Comment (peterbarker) [inline MAVProxy/modules/lib/param_help.py:53]: This is also essentially an error and should be emitted all the time - verbose or not

File: MAVProxy/modules/lib/param_help.py [hunk 2]
```diff
@@ -31,26 +31,30 @@ def param_help_download(self):
...
                 return None
         if not os.path.exists(path):
-            print("Param XML (%s) does not exist" % path)
+            if verbose:
+                print("Param XML (%s) does not exist" % path)
```
Comment (peterbarker) [inline MAVProxy/modules/lib/param_help.py:57]: Also an error and should be displayed verbose-or-not

**Review by peterbarker — COMMENTED** [review]
Pretty sure we want to display out-and-out errors even if we're not verbose.

**Comment by peterbarker** [comment]
Nudge.  Want this feature, but also want the diagnostics even if we're not verbose.

File: MAVProxy/modules/lib/param_help.py [hunk 3]
```diff
@@ -31,26 +31,30 @@ def param_help_download(self):
...
         if self.last_pair == (self.xml_filepath, self.vehicle_name):
             return self.last_htree
         if self.xml_filepath is not None:
-            print("param: using xml_filepath=%s" % self.xml_filepath)
+            if verbose:
+                print("param: using xml_filepath=%s" % self.xml_filepath)
             path = self.xml_filepath
         else:
             if self.vehicle_name is None:
-                print("Unknown vehicle type")
```
[the remark this answers is not in the document]
Comment (tridge) [reply MAVProxy/modules/lib/param_help.py:43]: nope, this does happen and would be very annoying in set and show

File: MAVProxy/modules/lib/param_help.py (hunk 1 above)
Comment (tridge) [reply MAVProxy/modules/lib/param_help.py:53]: nope, it would be extremely annoying.

File: MAVProxy/modules/lib/param_help.py (hunk 2 above)
Comment (tridge) [reply MAVProxy/modules/lib/param_help.py:57]: nope

**Pull request merged**